lib/config/scheme.js

```diff
@@ -0,0 +1,24 @@
+const configParser = require('gemini-configparser');
+
+const root = configParser.root;
+const section = configParser.section;
+const option = configParser.option;
+
+const shouldBeString = require('../metadata/validation-utils').shouldBeString;
+
+const ENV_PREFIX = `${require('../../package').name}_`;
+
+module.exports = root(
+    section({
+        username: option({
+            validate: shouldBeString
+        }),
+
+        apiToken: option({
+            validate: shouldBeString
+        }),
+
+        url: option({
+            validate: shouldBeString
+        })
+    }), {envPrefix: ENV_PREFIX});
```

Comment:
`parseEnv` is not intended for searching required env variables. It should be used to parse non-trivial values such as JSON entries or something like screen_resolution=1024x768. 
But you're right, it may worth to add comment there